core/mutation/mutation.go

```diff
@@ -28,6 +28,7 @@ import (
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 
+	spb "github.com/google/keytransparency/impl/proto/mutation_v1_service"
```

Comment:
OK, `RegisterMutationServiceHandlerFromEndpoint` was missing. That's why it didn't work.